.gitignore

```diff
@@ -0,0 +1,16 @@
+# directories
+venv*/
+notebooks/
```

Comment:
not really. This gitignore should not be pushed (i.e ignore your gitignore which defines specific things for your environment). I think he has that because I use this in all on all my workspaces so it made sense to have it in my branch.